packages/playwright-core/src/server/socksClientCertificatesInterceptor.ts

```diff
@@ -27,6 +27,11 @@ import { SocksProxy } from '../common/socksProxy';
 import type * as channels from '@protocol/channels';
 import { debugLogger } from '../utils/debugLogger';
 
+const dummyServerCerts: tls.TlsOptions = {
```

Comment:
Maybe do it lazily? Otherwise we read certs unnecessary for 99.9% of our users.

Reply:
Fair point!